operations/boot_orchestration/Ascertain_Which_BOS_Session_Booted_A_Node.md

```diff
@@ -0,0 +1,87 @@
+# Ascertain Which BOS Session Booted a Node
```

Comment:
It might be better to pick a simpler verb instead of "ascertain". Perhaps "determine"?

Reply:
Aack, simpletons. ;)  I'll change ascertain to determine in the file and the filename.